README.md

```diff
@@ -42,4 +42,54 @@ pip install -r requirements.txt
 make html
 ```
 
-You can now open the Handbook website on `_build/html/index.html` in your browser. Make corresponding changes on the documentation site and then run `make clean && make html` to update the documentation. You can now create a pull request to get your changes merged into the upstream develop branch.
\ No newline at end of file
+You can now open the Handbook website on `_build/html/index.html` in your browser. Make corresponding changes on the documentation site and then run `make clean && make html` to update the documentation. You can now create a pull request to get your changes merged into the upstream develop branch.
+
+## Project Structure
+
```

Comment:
Can you add some placeholder text about how a contributor can use this to navigate the handbook?

Reply:
Yes will mention that too